src/lib/storage/column_iterables.hpp

```diff
@@ -80,6 +80,47 @@ class ColumnIterable {
     });
   }
 
+  /**
+   * @defgroup Function for the materialization of values and nulls. The following implementations may be overridden by
```

Comment:
I’m sorry Moritz. Could you put the last sentence on a new line, please? What comes after `@defgroup` is the name of the group. The rest is just additional information.

Reply:
sure